whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/domain/DomainIntersectionValidator.java

```diff
@@ -41,10 +45,11 @@ public DomainIntersectionValidator(final Ipv4DomainTree ipv4DomainTree,
     public void validate(final PreparedUpdate update, final UpdateContext updateContext) {
         final Domain domain = Domain.parse(update.getUpdatedObject().getKey());
         if (domain.getType() != INADDR) {
```

Comment:
There is already a NServerValidator . Is there a reason we are using intersection validator class for Nserver validation?

Reply:
I was thinking that we have a validator by each service but I saw yesterday that we can have a lot of them and NServerValidator is one of them in this case. So I moved there